site/app/templates/officeHoursQueue/QueueStatus.twig

```diff
@@ -96,6 +96,14 @@
       <input type="hidden" name="queue_code" value="{{ viewer.getCurrentQueueCode() }}"/>
       <button id="leave_queue" type="submit" class="btn btn-danger">Leave Queue</button>
     </form>
+      {% set my_queue = 'code' %}
+      {% set open_queues = viewer.getAllQueues()|filter(queue => queue['open'] == 'true') %}
```

Comment:
Is this something that should be handled on the view?

If the student should only ever interact with queues that are set to open then there is no reason to send the entire list of qs to the twig page.

I think adding a function like `getAllOpenQueues` to the model and  then filtering at the database level for open queues only is a better solution